index.html

```diff
@@ -1,88 +1,61 @@
     <title>Check Vaccine Status</title>
-    <style>
-        .output td {
-            border: 1px solid black;
-            text-align: center;
-        }
-
-        /* .center {
-            width: 70%;
-            margin: auto;
-            border: 1px solid #4287f5;
-
-        } */
-
-        h1,
-        h4 {
-            text-align: center;
-        }
-
-        #rightAlign {
-            text-align: right;
-            font-weight: bold;
-        }
-    </style>
-</head>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
 
-<body>
+  <body>
     <center>
-        <div class="center">
-            <h1>Vaccine Availability Status</h1>
-            <h4 id="refreshTimerText"></h4>
-            <table class="input">
-                <tbody>
-                    <tr>
-                        <td>Date [DD-MM-YYYY]</td>
-                        <td><input type="text" id="dateForCheckingAvailability" /></td>
-
-                    </tr>
-
-                    <tr>
-                        <td>Disctrict Code(s) [Comma Separated]</td>
-                        <td><input type="text" id="disctrictCodes" /></td>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td id="rightAlign">Press Enter to Search:</td>
-                        <td>
-                            <center> <button type="button" onclick="search();">Search</button> </center>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-            <p>
-                Options</br>
-                <input type="checkbox" id="optionsAge18plus" checked>Age 18+ &amp &lt 45 </input></br>
-                <input type="checkbox" id="optionsAge45plus">Age 45+ </input> </br>
-                <input type="checkbox" id="optionsDose1" checked>Dose 1 </input></br>
-                <input type="checkbox" id="optionsDose2">Dose 2 </input>
-            </p>
-            <h4 id="API_CALL_Count"></h4>
-
-
-            <table class="output" id="mainTable">
-                <tbody>
-                    <tr>
-                        <td>District Code</td>
-                        <td>District Name</td>
-                        <td>Availability</td>
-                        <td>Name</td>
-                        <td>Pin Code</td>
-                        <td>Available Capacity for Dose 1</td>
-                        <td>Available Capacity for Dose 2</td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
+      <div class="center">
+        <h1>Vaccine Availability Status</h1>
+        <h4 id="refreshTimerText"></h4>
+        <table class="input">
+          <tbody>
+            <tr>
+              <td>Date [DD-MM-YYYY]</td>
+              <td><input type="text" id="dateForCheckingAvailability" /></td>
+            </tr>
+
+            <tr>
+              <td>Disctrict Code</td>
```

Comment:
There are multiple district codes that can be given as input by users which are comma separated.
So we need to change "District _Code_" to "District _Codes_"
Also, when we remove "[Comma Separated]" from the <td>, how else are we notifying the user that the text field accepts comma separated values rather than space separated?
Currently we have 2 default values of 392 and 395 which are comma separated, so it becomes evident.
However, if we decide to change the default values from "392,395" to say "392", then user would not be aware that this app is capable of searching through multiple districts and not just a single district at a time.